alpha/lib/model/LiveEntryServerNode.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * @package Core
+ * @subpackage model
+ */
+class LiveEntryServerNode extends EntryServerNode{
```

Comment:
KalturaLiveEntryServerNode has only one param "parameters" there is a mismatch between the classes.